### ISSUE TITLE
Design a Mockup for ECU Listing UI

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#27]
## User story
1. As a user
2. I want to see a visual representation of the ECU Listing UI
3. So that I understand how to browse available ECUs, select them for flashing, and access detailed information about each one

## Acceptance criteria
* A clear visualization of the GUI layout and design is provided
* Available ECUs, selection of the needed ECU and details are included in the mockup
* User interaction flow for browsing available ECUs, selecting the needed ECU for flashing and accessing its information is reflected

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
